### PR TITLE
Add link to Github repo

### LINKED ISSUE #11: Add link to Github repo
There should be a link to the Github repo so users can find the repo.

### PR DESCRIPTION
**Pull request by vr-varad**
Used React Icons for add Github Icon
Resolves #11

### CODE REVIEW

**Comment by vr-varad** [comment]
@steezeburger

**Comment by vr-varad** [comment]
@steezeburger any updates?

**Comment by steezeburger** [comment]
@vr-varad sorry! I was away camping. I will test this tonight!

File: web/src/components/SearchBar/SearchBar.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React, { memo, useState } from "react";
+import { AiFillGithub } from "react-icons/ai";
```
Comment (steezeburger) [inline web/src/components/SearchBar/SearchBar.tsx:2]: Does this `react-icons` package need to be added to `package.json`?

**Comment by steezeburger** [comment]
@vr-varad I apologize for the delay. I have been very busy with work!

**Comment by vr-varad** [comment]
No problem!! Just wanted to ask xan I keep resolving the issue even if u are not present?
@steezeburger

File: web/src/components/SearchBar/SearchBar.tsx (hunk 1 above)
Comment (steezeburger) [reply web/src/components/SearchBar/SearchBar.tsx:2]: @vr-varad

**Comment by steezeburger** [comment]
@vr-varad the issue shouldn't be resolved till the fix is merged, which requires my approval for now.

**Pull request closed without merging**